README.md

```diff
@@ -20,7 +21,9 @@ Before using `permaweb-deploy`, you must:
    ```bash
    base64 -i wallet.json | pbcopy
    ```
-3. Ensure that the secret name for the encoded wallet is `DEPLOY_KEY`.
+2. Ensure that the secret name for the encoded wallet is `DEPLOY_KEY`.
```

Comment:
Just a thought here, would be preferable to store as json string. B64 encoding doesnt actually gain any protection - its still a string, just like a json string, and requires more work to decode it.

Reply:
And considering eth and pol wallets dont use it... well it seems even stranger to still have this.

Reply:
True, but updating the required method for providing an Arweave wallet's jwk would be a major breaking change. It should be discussed in more detail with the repo owner directly before any community prs are made to address it.

Reply:
@twilson63 wdyt?